geniza/corpus/management/commands/import_data.py

```diff
@@ -64,6 +64,19 @@
         "Editor(s)": "editor",
         "Translator (optional)": "translator",
     },
+    "demerged": {
```

Comment:
Aren't these all the same as the metadata fields? Can't we use the same list?

Reply:
Could specify via an optional param to `get_csv` to override the field config used; by default it would use the data name